utah_bike_demand_model.py

```diff
@@ -85,6 +88,7 @@ def preprocess_network(net):
             slope = net.get_link_attribute_value((a,b), 'slope')
             bike_blvd = net.get_link_attribute_value((a,b), 'bike_boulevard')
             bike_path = net.get_link_attribute_value((a,b), 'bike_path')
+            bike_lane = net.get_link_attribute_value((a,b), 'bike_lane') # New
             no_bike_lane = not net.get_link_attribute_value((a,b), 'bike_lane')
```

Comment:
this line can now be `no_bike_lane = not bike_lane`

Reply:
went with this option